libraries/mbed/targets/hal/TARGET_WIZNET/TARGET_W7500x/rtc_api.c

```diff
@@ -0,0 +1,78 @@
+/* mbed Microcontroller Library
+ * Copyright (c) 2006-2013 ARM Limited
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+#include "rtc_api.h"
+
+time_t wiz_rtc_time;
+char rtc_enabled = 0;
+
+#ifdef __cplusplus
+extern "C"{
+#endif
+void PWM3_Handler(void)
```

Comment:
Can you please use NVIC_SetVector rather then redefining weak IRQ symbol ? An example:  https://github.com/mbedmicro/mbed/blob/master/libraries/mbed/targets/hal/TARGET_NXP/TARGET_LPC176X/gpio_irq_api.c#L100

IT would become:
```
void rtc_handler(void)
{
     wiz_rtc_time++;
     PWM_CH3_ClearOverflowInt();
}

#in init:
NVIC_SetVector(PWM3_Handler, (uint32_t)rtc_handler);
```
There's again problem spaces/tabs here?